src/simulation/hdf_simulation_store.jl

```diff
@@ -451,6 +468,21 @@ end
 """
 Write an emulation model result for a timestamp to the store.
```

Comment:
Is this docstring still correct? My understanding is that the timestamp is the simulation time, not necessarily the timestamp for the result which correlates to the index.